### PR TITLE
Dan f97/redux shape

### PR DESCRIPTION
**Pull request by dnlflores**
commented out old state shape example for projects students no longer use. Also kept the old generic Soundcloud, and Meetup for later when Airbnb has been fully polished.

### CODE REVIEW

File: assets/react-redux-state-shape.md [hunk 1]
```diff
@@ -16,39 +16,140 @@
...
+        // Refer to your API Docs to get more information.
+        allSpots: {
+            [spotId]: {
+                spotData
+            },
+            // These optional ordered lists are for you to be able to store an order in which you want your data displayed.
+            // you can do this on the frontend instead of in your slice is state which is why it is optional.
+            optionalOrderedList: []
+        },
+        // Notice singleSpot has more data that the allSpots slice. Review your API Docs for more information.
```
Comment (alex-pezzati) [inline assets/react-redux-state-shape.md:90]: Looks like this comment is meant for the "Beginner" redux state example.

**Review by alex-pezzati — CHANGES_REQUESTED** [review]
This overall looks really great.  Do you think we should have similar comments on the "Beginner" section as well?

- I'd like to see this structure applied to the MeetUp and SoundCloud examples.
- For all of the code that is commented out, we could move that to the bottom of the file and make a note in the comments that it is not currently used.

**Comment by dnlflores** [comment]
I added comments to beginner section for review

**Review by alex-pezzati — APPROVED** [review]
Getting this in for now. We'll make revisions soon.

**Pull request merged**